### PR TITLE
USHIFT-1724: Add runner and minimal conformance test list

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: origin/run.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+DEST_DIR="${DEST_DIR:-${DEFAULT_DEST_DIR}}"
+[ -d "${DEST_DIR}" ] || mkdir -p "${DEST_DIR}"
+DEST_DIR="$(realpath "${DEST_DIR}")"
+
+cd "${SCRIPT_DIR}"
+
+echo "Building openshift-tests"
+make build
+
+if ! which oc; then
```
Comment (ggiguash) [inline origin/run.sh:16]:
```suggestion
if ! which oc &>/dev/null; then
```

File: origin/run.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    echo "oc binary not found"
+    exit 1
+fi
+
+if ! which kubectl; then
```
Comment (ggiguash) [inline origin/run.sh:21]:
```suggestion
if ! which kubectl &>/dev/null; then
```

File: origin/run.sh (hunk 1 above)
Comment (pacevedom) [reply origin/run.sh]: Done

File: origin/run.sh (hunk 2 above)
Comment (pacevedom) [reply origin/run.sh]: Done

**Comment by dhellmann** [comment]
/hold

I'd like to set the new job using this script pass before we merge the script. We talked about doing that by having the job be "smart" and only run if it found the runner script. We could also do it by making the job optional and invoking it explicitly on this PR.

**Comment by pacevedom** [comment]
/override ci/prow/microshift-metal-tests
This change does not impact any tests yet.

**Comment by pacevedom** [comment]
> /hold
> 
> I'd like to set the new job using this script pass before we merge the script. We talked about doing that by having the job be "smart" and only run if it found the runner script. We could also do it by making the job optional and invoking it explicitly on this PR.

Didn't see this comment. PR for that coming in today, I already got it working in a draft.

**Comment by pacevedom** [comment]
Waiting for https://github.com/openshift/release/pull/44372

**Comment by pacevedom** [comment]
Conformance tests are now passing.

**Pull request merged**